### DISCUSSION TITLE
Change btn color with sass

### DISCUSSION THREAD

**Discussion by VictorStochero** [bootscore/bootscore#351] [Q&A] [answered]
I'm trying to change the colors of the primary button, I managed to change the background when using: `$primary: #A4D361;`
the css looked like this on the page:
```
.btn-primary {
    --bs-btn-color: #000;
    --bs-btn-bg: #A4D361;
    --bs-btn-border-color: #A4D361;
    --bs-btn-hover-color: #000;
    --bs-btn-hover-bg: #b2da79;
    --bs-btn-hover-border-color: #add771;
    --bs-btn-focus-shadow-rgb: 139, 179, 82;
    --bs-btn-active-color: #000;
    --bs-btn-active-bg: #b6dc81;
    --bs-btn-active-border-color: #add771;
    --bs-btn-active-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
    --bs-btn-disabled-color: #000;
    --bs-btn-disabled-bg: #A4D361;
    --bs-btn-disabled-border-color: #A4D361;
}
```
but I couldn't change the color of the text, I tried several variables that I found in the [Bootstrap Docs](https://getbootstrap.com/docs/5.3/components/buttons/#sass-variables) like `$btn-link-color:   #005EAC;` and `$btn-link-hover-color: #005EAC;`.

**Comment by justinkruit** [comment, marked as answer]
To quickly and easily override the text color on the primary button, you can just change the css variables by adding a new class in your own styling.
```
.btn-primary {
  --bs-btn-color: #fff;
}
```
The default colors are calculated by Bootstrap while compiling by checking the contrast of the background color and then giving either a black or white color.
